usr/share/rear/finalize/Linux-i386/670_run_efibootmgr.sh

```diff
@@ -19,60 +19,60 @@ is_true $EFI_STUB && return
 test -f "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" || return 0
 
 # Determine where the EFI System Partition (ESP) is mounted in the currently running recovery system:
-esp_mountpoint=$( df -P "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" | tail -1 | awk '{print $6}' )
-# Use TARGET_FS_ROOT/boot/efi as fallback ESP mountpoint:
-test "$esp_mountpoint" || esp_mountpoint="$TARGET_FS_ROOT/boot/efi"
+esp_mountpoint=$( filesystem_name "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" )
+# Use TARGET_FS_ROOT/boot/efi as fallback ESP mountpoint (filesystem_name returns "/"
+# if mountpoint not found otherwise):
+if [ "$esp_mountpoint" = "/" ] ; then
+    esp_mountpoint="$TARGET_FS_ROOT/boot/efi"
+    Log "Mountpoint of $TARGET_FS_ROOT/$UEFI_BOOTLOADER not found, trying $esp_mountpoint"
```

Comment:
Could you use `DebugPrint` (which also logs) instead of only `Log`
so that the message also appears on the user's terminal
when he runs recover in debug mode.
At least I prefer such messages where ReaR has to make special decisions
to also appear on the terminal because there it is easier to notice them
(instead of having to actively search for them from the log file).

Reply:
@jsmeix but `DebugPrint` also logs only when DEBUG is set, no? So if the user has not used debug mode, the message will go neither to the log not to the terminal. This does not seem right, because if we got here, we may encounter a potential problem later and the message in the log will be useful for debugging what has happened.

Reply:
We would need something like `LogDebugPrint`, but this function does not exist. I could use `LogPrint`, which would cause the message to go to log always and to terminal if verbose is set (instead of if debug is set). Would that be acceptable?

Reply:
In general I perfer too much verbosity over too less verbosity in verbose mode
(the former can be simply ignored, the latter leaves the user in the dark)
so yes, please use `LogPrint`.

Note that I wrote "in verbose mode" so by default ReaR should work silent
except real error messages but if ReaR's verbose mode is a bit too verbose
it is still better that too less verbose.

Reply:
I changed all `Log` to `LogPrint` in the fallback branches.